main.go

```diff
@@ -820,7 +813,10 @@ to get your bearings.
 		}
 
 		v := connect()
-		if opt.Move.Recurse {
+
+		//Don't try to recurse if operating on a key
+		// args[0] is the source path. args[1] is the destination path.
+		if opt.Move.Recurse && !vault.PathHasKey(args[0]) && !vault.PathHasKey(args[1]) {
```

Comment:
What that line does is avoids passing it to the recursion code that has no concept of key granularity. It works as expected, but the code handling it is the non-recursive code.